charts/scalardl/templates/ledger/deployment.yaml

```diff
@@ -114,13 +144,27 @@ spec:
               command:
               - /usr/local/bin/grpc_health_probe
               - -addr=:50051
+              {{- if .Values.ledger.tls.enabled }}
+              - -tls
+              {{- if .Values.ledger.tls.caRootCertSecret }}
+              - -tls-ca-cert=/tls/certs/ca-root-cert.pem
+              {{- end }}
+              - -tls-server-name={{ .Values.ledger.tls.overrideAuthority }}
```

Comment:
Ditto. (The same question as the ScalarDB Cluster PR.)

Reply:
Thank you for your review!
I answered it on the ScalarDB Cluster PR side.
https://github.com/scalar-labs/helm-charts/pull/255#discussion_r1535123060